frontend/src/app/testResults/types.d.ts

```diff
@@ -0,0 +1,19 @@
+type MultiplexDisease = "COVID-19" | "Flu A" | "Flu B";
+type TestResult = "POSITIVE" | "NEGATIVE" | "UNDETERMINED" | "UNKNOWN";
+
+interface DiseaseName {
+  disease: {
+    name: MultiplexDisease;
+  };
+}
+
+interface SRMultiplexResult extends DiseaseName {
```

Comment:
⚠️ Need to have these two different types `SRMultiplexResult` and `PxpMultiplexResult` because of https://github.com/CDCgov/prime-simplereport/pull/3920#discussion_r911505600